client/src/index.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import Carousel from './components/Carousel';
+import Carousel from './components/carousel';
```

Comment:
I'm going to stick with lowercased for now. I originally wanted to go uppercase as you mentioned for React style, but when typing the path out, it auto selects it as lowercase for some odd reason, and to prevent potential incorrect path bugs in the future, going to make all files lowercased for now.